V2/gameFuse.js

```diff
@@ -155,7 +156,7 @@
     }
 
     static signIn(email, password, callback = undefined) {
-        this.Instance.signInPrivate(email, password, callback);
+        return this.Instance.signInPrivate(email, password, callback);
```

Comment:
This change was in order to make sure that the caller has access to the Promise that gets returned.